src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static Properties getProperties() {
+        Properties properties = new Properties();
+        try (FileInputStream input = new FileInputStream("src/main/resources/config.properties")) {
+            properties.load(input);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read credentials from file", e);
+        }
+        return properties;
+
+    }
+
+    public static Connection getConnection() {
+        Properties properties = getProperties();
+        String dbUrl = properties.getProperty("db.url");
+        String dbUsername = properties.getProperty("db.username");
+        String dbPassword = properties.getProperty("db.password");
+        try {
+            return DriverManager.getConnection(dbUrl, dbUsername, dbPassword);
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create connection", e);
```

Comment:
Consider using a more specific exception type instead of `RuntimeException`. You could use a custom exception like `DataProcessingException` to provide more context about the error.